### DISCUSSION TITLE
反馈个问题

### DISCUSSION THREAD

**Discussion by sxwu** [XTLS/Xray-core#1987] [General]
使用xtld-rprx-vision,  我访问我的webmin的页面会出错。关掉立马好了。

**Comment by GreatMichaelLee** [comment]
你这么反馈问题，除了我，没人会理你的

**Comment by sxwu** [comment]
不好意思，我比较业余。
我在使用vless+tls+xtls-rprx-vision 时，打开我的服务器webmin面板会直接跳页面出错。其他网站没遇到问题，
我只是反映一个使用中遇到的问题。
把flow设置成空就没正常了了。我想应该时vision的问题。


从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: michael ***@***.***>
发送时间: 2023年4月25日 23:08
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [XTLS/Xray-core] 反馈个问题 (Discussion #1987)


你这么反馈问题，除了我，没人会理你的

―
